docs/Using.md

```diff
@@ -34,6 +34,12 @@ If you do not provide this value, `hipcc` will check for existance of the follow
 
 Preserves runtime temporary compilation files when this variable is set to `1`.
 
+#### CHIP\_LAZY\_JIT
```

Comment:
Are the underscores needed?

Reply:
Markdown processors may interpret the underscores, so probably yes ([link](https://www.markdownguide.org/basic-syntax/#italic-best-practices)).